### ISSUE TITLE
Raycasting processors have normalized isovalue

### ISSUE THREAD

**Issue by martinfalk** [inviwo/inviwo#58]
All raycasting processors use a normalized parameter [0,1] to select the isovalue. The TF editor as well as the `Surface Extraction` use the regular, unnormalized data range.

**Comment by petersteneteg** [comment]
well the TF knows nothing about real values either... it's all 0 to 1 there to..  the data values there is just a gui thing...

**Comment by martinfalk** [comment]
yes, and exactly here it is sensible to have "proper" values. Think of linking to isosurface render processors for different volumes but the same isovalue.

**Comment by martinfalk** [comment]
Right now, I need a calculator for that... besides the fact that I see 9.8 in the TF editor and have to convert it from [3.5, 914.1] to [0, 1] to get the corresponding isovalue.

**Comment by petersteneteg** [comment]
Think having real values in the TF was one of Tinos main issues...
